src/Randomizer.SMZ3/Regions/SuperMetroid/Maridia/InnerMaridia.cs

```diff
@@ -183,22 +184,27 @@ public LeftSandPitRoom(InnerMaridia region)
                     name: "Left",
                     alsoKnownAs: new[] { "Missile (left Maridia sand pit room)" },
                     vanillaItem: ItemType.Missile,
-                    access: items => region.CanReachAqueduct(items) && items.Super && Logic.CanPassBombPassages(items) && Logic.CanNavigateMaridiaLeftSandPit(items),
+                    access: items => CanEnter(items) && (Logic.CanWallJump(WallJumpDifficulty.Easy) || Logic.CanNavigateMaridiaLeftSandPit(items)), //! Double check the logic here
```

Comment:
Here's the Left Sand Pit checks. Not sure if I handled this correctly.

Reply:
So the wall jumping here that Pink pictures is actually independent of the final wall jump. That being said, I'd contest this being an "easy" wall jump room as the first jump is out of the sand which can be pretty precise and annoying. It's easy in the sense that there's not a huge series, but it can be very frustrating when you fall back into the sand.

Reply:
I've changed it to medium, or easy with spring ball. Having the option to require spring ball on just requires spring ball and hi jump.

I know Hi Jump fits into this otherwise, I'm just not sure how. Maybe this could use a final check, otherwise I guess I'll leave it like this.